Makefile

```diff
@@ -57,17 +65,37 @@ run-local: install-apis ## Install CRDs and run provisioner locally
 	kubectl create namespace rukpak-system
 	$(Q)go run provisioner/k8s/main.go
 
+deploy: install-apis ## Deploy the operator to the current cluster
+	kubectl apply -f provisioner/k8s/manifests
+
+run: build-local-container load-image deploy
+
+## --------------------------------------
+## Build and Load
+## --------------------------------------
+
 # Binary builds
-GO_BUILD := $(Q)go build
+.PHONY: build bin/k8s bin/unpack
 
-build: bin/k8s
+GO_BUILD := $(Q)go build
+build: bin/k8s bin/unpack
 
 bin/k8s:
 	CGO_ENABLED=0 go build -o $@ ./provisioner/k8s
 
 bin/unpack:
 	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
 
+build-local-container: ## Builds provisioner container image locally
+	docker build -f Dockerfile -t $(IMAGE) .
+
+kind-load: ## Load-image loads the currently constructed image onto the cluster
+	 ${KIND} load docker-image $(IMAGE)
+
+kind-cluster: ## Standup a kind cluster for e2e testing usage
+	${KIND} get clusters | grep ${KIND_CLUSTER_NAME} || ${KIND} create cluster --name ${KIND_CLUSTER_NAME}
```

Comment:
Once upon a time (and maybe this is still the case), I had the operator-sdk e2e setup to automatically teardown the e2e cluster as part of the run to ensure the cluster didn't retain state between runs.

I'm fuzzy on the details, but I think I had it setup to tear it down after successful completion so that if the e2e failed, the cluster would still be there for inspection.